README.md

```diff
@@ -10,14 +10,35 @@ With Contentful and Gatsby you can connect your favorite static site generator w
 
 ## Features
 
-* Simple content model and structure. Easy to adjust to your needs.
-* Use the [synchronization feature](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/synchronization) of our [Delivery API](https://www.contentful.com/developers/docs/references/content-delivery-api/).
-* Responsive/adaptive images via [gatsby-image](https://www.gatsbyjs.org/packages/gatsby-image/) and our [Images API](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/synchronization/initial-synchronization-of-entries-of-a-specific-content-type).
+- Simple content model and structure. Easy to adjust to your needs.
+- Use the [synchronization feature](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/synchronization) of our [Delivery API](https://www.contentful.com/developers/docs/references/content-delivery-api/).
+- Responsive/adaptive images via [gatsby-image](https://www.gatsbyjs.org/packages/gatsby-image/) and our [Images API](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/synchronization/initial-synchronization-of-entries-of-a-specific-content-type).
 
 ## Getting started
 
 See our [official Contentful getting started guide](https://www.contentful.com/developers/docs/tutorials/general/get-started/).
 
+### Get the source code and install dependencies.
+
+```
+$ git clone https://github.com/contentful/starter-gatsby-blog.git
+$ npm install
+```
+
+Or use the [Gatsby CLI](https://www.npmjs.com/package/gatsby-cli).
+
+```
+$ gatsby new contentful-starter-blog https://github.com/contentful/starter-gatsby-blog/
+```
+
+### Set up of the needed content model and create a configuration file
+
+This project comes with a Contentful setup command `npm run setup`.
+
+This command will ask you for a space ID, and access tokens for the Contentful Management and Delivery API and then import the needed content model into the space you define and write a config file (`./contentful.json`).
```

Comment:
I think this was wrong from the original README. I think it should be this:

```suggestion
This command will ask you for a space ID, and access tokens for the Contentful Management and Delivery API and then import the needed content model into the space you define and write a config file (`./.contentful.json`).
```

Reply:
In fact, after looking at how the setup works, I don't think this is even true -- it writes to `.env` files, not `.contentful` files. We don't need to look at this now, but I think we should follow up on this since it doesn't seem to be accurate (although I don't have enough insight into how Gatsby works to know if it implicitly reads the `.contentful.json` file).